packages/components/src/flex/flex/README.md

```diff
@@ -25,38 +25,44 @@ function Example() {
 
 ## Props
 
-##### align
-
-**Type**: `CSSProperties['alignItems']`
+### `align`: `CSSProperties['alignItems']`
 
 Aligns children using CSS Flexbox `align-items`. Vertically aligns content if the `direction` is `row`, or horizontally aligns content if the `direction` is `column`.
 
-##### direction
+-   Required: No
+-   Default: `center`
 
-**Type**: `[ResponsiveCSSValue<CSSProperties['flexDirection']>]`
+### `direction`: `ResponsiveCSSValue<CSSProperties['flexDirection']>`
 
 The direction flow of the children content can be adjusted with `direction`. `column` will align children vertically and `row` will align children horizontally.
 
-##### expanded
+-   Required: No
+-   Default: `row`
 
-**Type**: `[boolean]`
+### `expanded`: `boolean`
 
 Expands to the maximum available width (if horizontal) or height (if vertical).
 
-##### gap
+-   Required: No
+-   Default: `true`
 
-**Type**: `[number | string]`
+### `gap`: `number`
```

Comment:
This is another change that we may consider: I personally find it confusing that we exposed `align` and `justify` behaving exactly like the CSS properties, while we use an internal grid system for `gap` (which is also [a valid CSS prop for flex layouts](https://css-tricks.com/almanac/properties/g/gap/)) 

@mirka what are your thoughts on this one?

Reply:
This is a problem with `SpaceInput` values in general, but I agree that the polymorphic nature of those values can be confusing. (Especially since React `style` props interpret unitless `number` values to be pixels by default!) Unfortunately I can't think of an easy way around it, other than removing SpaceInput support from the components themselves and make the consumer use `space()` from the outside, if they need to.

As an aside, I believe the [internal `gap`-to-margin implementation](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/flex/flex/hook.js) we have right now was kind of meant as a polyfill, since Safari didn't support flex gap when the `Flex` component was first written. I don't know if there are active bugs in there, but we can look into removing it in a separate issue. It would be nice to simply rely on native CSS gap.

Reply:
>This is a problem with SpaceInput values in general, but I agree that the polymorphic nature of those values can be confusing. (Especially since React style props interpret unitless number values to be pixels by default!) Unfortunately I can't think of an easy way around it, other than removing SpaceInput support from the components themselves and make the consumer use space() from the outside, if they need to.

Yeah, this may require some more investigation

> As an aside, I believe the [internal gap-to-margin implementation](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/flex/flex/hook.js) we have right now was kind of meant as a polyfill, since Safari didn't support flex gap when the Flex component was first written. I don't know if there are active bugs in there, but we can look into removing it in a separate issue. It would be nice to simply rely on native CSS gap.

I get the same feeling — it would be definitely great to rely on native CSS though

Reply:
Yes. I would be great exploring if the native gap can be used. I guess its out of scope for this PR tough.